### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kaysabelle**
Still in progress! Open for feedback while I finish writing it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,101 @@
-radlib-cordova
-===============
+RadLib
+=======================================================
+An Apache Cordova plugin that allows for interfacing mobile applications with RFID tag readers.
+######The currently supported readers are as follows:
+- [Technology Solutions 1128 Bluetooth UHF Reader](http://www.tsl.uk.com/products/1128-bluetooth-handheld-uhf-rfid-reader/)
+- RC522 RF Reader()
```
Comment (kaysabelle) [inline README.md]: Need a link for this @MinickLS

File: README.md [hunk 2]
```diff
@@ -1,4 +1,101 @@
...
+        <li>model: model of the reader device (only required for "BLUETOOTH" connectionType). Current accepted models are "ARDUINO_RC522_HF" or "TSL_1128_UHF"</li>
+        <li>address: address of the reader device (only required for "BLUETOOTH" connectionType)</li>
+        <li>connectionType: type of connection for device. Currently only "BLUETOOTH" and "CAMERA" are supported</li>
+      </ul>
+      Note: the required properties for this reader object are similar to what is returned in <code>scan()</code>. It is possible to save the object obtained from <code>scan()</code>, add a model property to it, and then use it for <code>connect()</code>.
+    </td>
+  </tr>
+</table>
+
+
```
Comment (jeslin) [inline README.md:122]: add details for bluetooth utils, and RC522 reader link

File: README.md [hunk 3]
```diff
@@ -1,4 +1,164 @@
...
+  </tr>
+</table>
+
+<table>
+  <tr>
+    <td colspan="2"><code>radlib.connect = function (success, failure, reader);</code></td>
+  </tr>
+  <tr>
+    <td colspan="2">Description:</td>
+  </tr>
```
Comment (jeslin) [inline README.md:66]: Add descriptions for connect and scan

File: README.md [hunk 4]
```diff
@@ -1,4 +1,164 @@
...
+      Callback function providing a success message
+    </td>
+  </tr>
+  <tr>
+    <td><code>failure</code></td>
+    <td>
+      Callback function providing a failure message
+    </td>
+  </tr>
+</table>
```
Comment (jeslin) [inline README.md:164]: :money_with_wings:

**Pull request merged**